db/migrate/20170221234419_create_accounting_transactions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAccountingTransactions < ActiveRecord::Migration
+  def change
+    create_table :accounting_transactions do |t|
+      t.string :qb_transaction_id, null: false
+      t.string :qb_transaction_type, null: false
```

Comment:
These should have indices, no? Separate, I think, not compound, because they are queried separately.